cmd/object_copy.go

```diff
@@ -78,10 +79,12 @@ func copyMain(cmd *cobra.Command, args []string) {
 
 	client.ObjectClientOptions.Version = version
 
-	if val, err := cmd.Flags().GetBool("debug"); err == nil && val {
-		setLogging(log.DebugLevel)
-	} else {
-		setLogging(log.ErrorLevel)
+	if execName == "stashcp" {
```

Comment:
Nope! From running on my end (at least for now, still needs some fixes) all modes work with the `-d` flag. Just added that for `stashcp` in `object_copy` since it does not go through `root.go`. Within `root.go` I check for the `-d` flag before looking for the setting in the config file.